tests/cnt-i18n_xhtml-lang/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" lang="fr">
+  <head>
+    <title lang="en">Processing the 'lang' attribute in XHTML content</title>
+    <meta charset="utf-8"/>
+  </head>
+  <body>
+    <p lang="en">Test passes if the following quotation is surrounded by the pair of '«' and '»' characters (used in French), instead of the '“' and '”' quotation marks (used in English).</p>
```

Comment:
```suggestion
    <p lang="en">Test passes if the following quotation is surrounded by a pair of '«' and '»' characters (used in French), instead of the '“' and '”' quotation marks (used in English).</p>
```